From 3515cd08d84d1f142b1c7d73246535d184a71841 Mon Sep 17 00:00:00 2001
From: Weiwen Chen <cww@rock-chips.com>
Date: Thu, 14 May 2020 20:05:28 +0800
Subject: [PATCH] ARM: dts: rockchip: Add customized definition parameter for
 rv1126-evb-ddr3-v10-tb

Add rk.root2nd to mount second rootfs.
For example:
rk.root2nd=/dev/mmcblk0p7 or rk.root2nd=/dev/mmcblk0p7.ext4

Signed-off-by: Weiwen Chen <cww@rock-chips.com>
Change-Id: I976700bc86e44fcef31ebbe8796e3d8968234024
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
index e0c45f7440a2..62358c6894fc 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
@@ -10,7 +10,7 @@
 	compatible = "rockchip,rv1126-evb-ddr3-v10-tb", "rockchip,rv1126";
 
 	chosen {
-		bootargs = "loglevel=0 initcall_nr_threads=-1 initcall_debug=0 printk.devkmsg=on root=/dev/rd0 console=ttyFIQ0 snd_aloop.index=7";
+		bootargs = "loglevel=0 initcall_nr_threads=-1 initcall_debug=0 printk.devkmsg=on root=/dev/rd0 rk.root2nd=/dev/mmcblk0p7 console=ttyFIQ0 snd_aloop.index=7";
 	};
 
 	memory {
-- 
2.35.3

